src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,74 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+import ReactNode = React.ReactNode
+import { UIComponent, childrenExist, customPropTypes, IRenderResultConfig } from '../../lib'
+
+/**
+ * A grid.
+ * @accessibility This is example usage of the accessibility tag.
+ * This should be replaced with the actual description after the PR is merged
+ */
+class Grid extends UIComponent<any, any> {
+  public static displayName = 'Grid'
+
+  public static className = 'ui-grid'
+
+  public static propTypes = {
+    /** An element type to render as (string or function). */
+    as: customPropTypes.as,
+
+    /** Primary content. */
+    children: PropTypes.node,
+
+    /** Additional classes. */
+    className: PropTypes.string,
+
+    /** The columns of the grid with a space-separated list of values. The values represent the track size, and the space between them represents the grid line. */
+    columns: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+    /** Shorthand for primary content. */
+    content: customPropTypes.every([
+      customPropTypes.disallow(['children']),
+      PropTypes.oneOfType([
+        PropTypes.arrayOf(customPropTypes.itemShorthand),
+        customPropTypes.itemShorthand,
+      ]),
+    ]),
```

Comment:
I agree with the reasons, but we have following things preventing us to do so:
- all components should support shorthand version; removing `content` breaks that pattern basically; it'd be the first component without shorthand support, right?
- we were discussing introducing `GridItem` component for controlling individual items; maybe it'll make sense at that point

Reply:
Yes, good points.  We probably should not make those types of decisions in this PR.  Let's keep it as is and we can discuss it in another.  This would be a framework level design change.